x/mongo/driver/operation.go

```diff
@@ -1103,6 +1104,85 @@ func (op Operation) addBatchArray(dst []byte) []byte {
 	return dst
 }
 
+func (op Operation) createLegacyHandshakeWireMessage(
```

Comment:
This is a direct copy and paste of the code removed in this commit: https://github.com/mongodb/mongo-go-driver/commit/44496173f8f03c24842511790fb99df4badf901b